### DISCUSSION TITLE
Can the search show only the description?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#382] [Ask a question] [answered]
Is ist possible to configure the search plugin to show only the `description` instead of the first n characters of the page?

**Reason for this question**

I am preparing a website that contains some html code at the begining of some pages. This html code is shown in the search results because they are at the beginning of the page.

The presentation of the page description instead of content might be a more useful solution for blogs or a wiki in some cases because it may contain an extract of the content instead possibly meaningless page preamble. 

This shall lead to a (public) template combining blog and wiki. It shall have a low footprint (no external libraries a.s.o.) for fast loading in any device especially mobile connections with low bandwith.

**Comment by schulle4u** [comment, marked as answer]
Have a look at line 16 in your `system/themes/snippets/content-search.php`, it defines the output of page description in search results. 

```
<div class="entry-content"><?php echo htmlspecialchars($yellow->toolbox->createTextDescription($page->getContent(false, 4096), $yellow->config->get("searchPageLength"))) ?></div>
```

You can output whatever page metadata you like. The following should work if you simply want to output your `Description:` page setting: 

```
<div class="entry-content"><?php echo $page->getHtml("description") ?></div>
```

Steffen

**Comment by BsNoSi** [comment]
Thanks Steffen. I am annoyed about the fact, that I have not enought time to get deeper into YELLOW faster. As always a very well described hint to customize a YELLOW-driven site. 

 As a result I think I should have a closer look to the snippets for some of my ideas. :)
